test/fail_compilation/test16002.d

```diff
@@ -0,0 +1,3 @@
+module test.fail_compilation.test16002;
+
+static assert(is(import.nonexistent == module));
```

Comment:
Did you mean "imports" here? Please also verify the actual error message with `TEST_OUTPUT`.